.github/workflows/nodejs.yml

```diff
@@ -154,7 +154,7 @@ jobs:
       - name: Checkout Repo
         uses: actions/checkout@v2
         with:
-          ssh-key: ${{ secrets.MONGO_NAV_PULL_KEY }}
+          ssh-key: ${{ secrets.LEAFYGREEN_UI_MONGO_NAV_DEPLOY }}
```

Comment:
May be worth adding a comment before each step explaining what's going on, for maintainability purposes

Reply:
What's unclear about "Checkout Repo"? We should improve the names if you feel it's unclear.

Reply:
Sorry not checkout repo specifically, but the other additional steps that you've added

Reply:
So "Set up SSH Agent" and "Push to Submodules"?